src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -283,7 +354,8 @@ public synchronized void setDrive(double throttle, double wheel, boolean quickTu
     if (quickTurn) {
       setOpenLoop(
           new DriveSignal(
-              (signal.getLeft() / scaling_factor) / 1.5, (signal.getRight() / scaling_factor) / 1.5));
+              (signal.getLeft() / scaling_factor) / 1.5,
```

Comment:
What's the 1.5? Change to a constant to make it more descriptive.

Reply:
This is @maxh65000's code.

Reply:
I didn't add this, a newline was just added for clarity.